Fix build with old clang by including the right header.
https://github.com/team-charls/charls/issues/62
https://github.com/team-charls/charls/commit/5a58c40f0fd38a75cbd3b1ac2d9dc6171a832f47
--- src/jpegls_error.cpp.orig	2019-12-29 13:10:31.000000000 -0600
+++ src/jpegls_error.cpp	2020-06-13 01:06:24.000000000 -0500
@@ -3,6 +3,8 @@
 
 #include <charls/jpegls_error.h>
 
+#include <string>
+
 using std::string;
 using std::error_category;
 
